framework/Simulation.py

```diff
@@ -149,236 +146,8 @@ def splitCommand(s):
     retList.append(buffer)
   return retList
 
-def createAndRunQSUB(simulation):
```

Comment:
This function has been abstracted to other places if I understand the changes I saw elsewhere, correct?